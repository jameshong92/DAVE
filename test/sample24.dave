// test table binding3
int main() {
	string s[] = {"a", "b", "c", "d", "e"};
	fld t1 = fld(s, "eng");
    int i[] = {1, 2, 3, 4, 5};
    fld t2 = fld(i, "num");
    fld[] b1 = {t1, t2};
    tbl test1 = tbl(b1);
    print(test1);
    string n[] = {"z", "y", "x", "w", "v"};
    fld t3 = fld(n, "rev");
    double n[] = {1.0, 2.0, 3.0, 4.0, 5.0};
    fld t4 = fld(n, "acc");
    fld[] b2 = {t3, t4};
    tbl test2 = tbl(b2);
    print(test2);
    tbl test = tbl(test1, test2, true);
    print(test);
}