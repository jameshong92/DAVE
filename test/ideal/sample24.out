eng	num	
a	1	
b	2	
c	3	
d	4	
e	5	

rev	acc	
z	1.1	
y	2.2	
x	3.3	
w	4.4	
v	5.5	

eng	num	rev	acc	
a	1	z	1.1	
b	2	y	2.2	
c	3	x	3.3	
d	4	w	4.4	
e	5	v	5.5	

